app/cypress/integration/cif/project-revision/index.spec.js

```diff
@@ -264,19 +266,20 @@ describe("the new project page", () => {
     cy.findByText(/Emissions Intensity Report/i).click();
     cy.findByText(/Add emissions intensity report/i).click();
     cy.url().should("include", "/form/6");
-
+    cy.intercept("POST", "http://localhost:3004/graphql", (req) => {
```

Comment:
Going forward, should this be added to all tests involving loading a page after a mutation?